tox.ini

```diff
@@ -198,3 +199,19 @@ commands =
 whitelist_externals =
     rm
     {[testenv]whitelist_externals}
+
+[testenv:changelog]
+description = Generate the changelog
+skip_install = False
```

Comment:
@webknjaz, this is required because of the way `towncrier` works, as far as I can tell. It makes an import of the package and if it cannot, it cannot find the version. The alternative is passing `--version=<VERSION>` using the `setuptools_scm` hack we have for the docker build tag but I'd rather avoid. This seems to be the simplest solution.

Reply:
ah... I see. OTOH supplying the version explicitly seems faster. It probably makes sense to post an FR for towncrier to allow specifying a callable...

Reply:
@decentral1se after thinking more about it I think that it's quite challenging to make it work with version autodiscovery.
We should generate a changelog before tagging git commit but then it'll auto-discover `.dev` version, not the one we're going to release.

So we should probably have a `release` tox env accepting the desired (next) version as a positional argument and
1) generating a changelog for that version
2) committing it
3) adding a Git tag to that commit

Reply:
Filed some ideas in upstream https://github.com/hawkowl/towncrier/issues/146

Reply:
Hmm, looks tricky. I don't think I'll have time to finish this.

Reply:
@decentral1se I've had a small chat with the maintainer of tox and virtualenv. They just don't care about shipping it in the dist now. They use draft mode and that's it. So maybe we could just do the same for now?